### PR TITLE
Update README.md to include other test requirements

### PR DESCRIPTION
**Pull request by ianromanick**
Knowing this in advance would have saved me some time.

### CODE REVIEW

**Comment by ianromanick** [comment]
It might be worth updating the "except" list so that we don't have to run the CI when just `README.md` is changed. :smile:

File: README.md [hunk 1]
```diff
@@ -236,6 +236,11 @@ Running `runtests` script-backed tests:
 cd $SOURCE_DIR/Test && ./runtests
 ```
 
+If some tests fail with validation errors, there may be a mismatch between the
+version of `spirv-val` on the system and the version of glslang.  In this
+case, it is necessary to run `update_glslang_sources.py`.  See "Check-Out
+External Projects" above for more details.
+
```
Comment (johnkslang) [inline README.md]: One should always run `update_glslang_sources.py` for many reasons before doing any new development.  It is part of the code base for HLSL legalization, validation, and testing.

Comment (johnkslang) [reply README.md]: If we are not saying this, we need to fix that; it's not just about validation.

File: README.md [hunk 2]
```diff
@@ -103,8 +103,8 @@ cd ../..
 ```
 
 If you wish to assure that SPIR-V generated from HLSL is legal for Vulkan,
-or wish to invoke -Os to reduce SPIR-V size from HLSL or GLSL, install
-spirv-tools with this:
+wish to invoke -Os to reduce SPIR-V size from HLSL or GLSL, or wish to run the
+integrated test suite, install spirv-tools with this:
```
Comment (johnkslang) [inline README.md]: Yes, okay.  These dependencies changed after the fact (comparatively recently).

**Comment by johnkslang** [comment]
Thanks.

**Pull request merged**